boxoffice/assets/js/views/edit_item_collection.js

```diff
@@ -0,0 +1,17 @@
+
+import {urlFor} from '../models/util.js';
+import {DetailView} from './detail_view.js';
+import {navigateTo} from '../views/main_admin.js';
+
+export const ItemCollectionEditView = {
+  render: function({ic_id}={}) {
+    DetailView.load({
+      url: urlFor('edit', {resource: 'ic', id: ic_id, root: true}),
+      title: 'Edit item collection',
+      handleForm: true,
+      onSuccess: function(responseData){
+        navigateTo(urlFor('view', {resource: 'ic', id: ic_id, root: true}))
```

Comment:
Could we also abstract `onSuccess` out and add it to DetailView

Reply:
Not necessarily, since `ItemCollectionEditView` (in this case) should decide what needs to happen after a form submit was successful eg: redirect to a particular route.